LWM2M_DevCapMgmt-v1_0_1.xml

```diff
@@ -15,17 +15,17 @@ NORMATIVE INFORMATION
 
   Information about this file can be found in the latest revision of
 
-    OMA-TS-LWM2M_DevCapMngt-V1_0
+    OMA-TS-LWM2M_DevCapMgmt-V1_0_3
 
-  This is available at http://www.openmobilealliance.org/
+  This is available at https://www.omaspecworks.org/
```

Comment:
omaspecworks.org in several places to refer to a TS location.

Reply:
Please see the previous related reply.

Please confirm if you wish the URL in these XML files and also in the template https://github.com/OpenMobileAlliance/OMA_Document_Templates/blob/master/SUP/n.xml to be changed.

Reply:
My view is that the link should point to: http://www.openmobilealliance.org/release/ 
This is generic enough and it can be setup in the template by default. If this is acceptable we will have to change the templates and rectify this pointer in all the objects that we try to rectify.

Reply:
@jpradocueva as requested the public URLs have been updated.